src/test/java/net/snowflake/client/core/SecureStorageManagerTest.java

```diff
@@ -213,6 +212,19 @@ Pointer getPointer() {
   private static final String idToken = "fakeIdToken";
   private static final String idToken0 = "fakeIdToken0";
 
+  @Ignore("This test won't run until we setup Mac & Windows testing env.")
```

Comment:
This is fine! We will have the test code ready for when we set up mac and windows testing.

Reply:
You mean we can just remove this Ignore line since the test is protected by platform judging if statement?

Reply:
See `ConditionalIgnoreRule` and `RunningOnTestaccount` to add `RunningWinMac` now. This way, when the tests can be enabled whenever a mac and win test start running and you won't forget to remove `@Ignore` annotation.

Reply:
Got it, great suggestion!